pypoller.py

```diff
@@ -114,12 +118,17 @@ def main(args):
 
                 time_t = round((end_t - start_t) * 1000, 2)
 
+                average = round((smoothing_factor * time_t) + (1 - smoothing_factor) * average, 2)
+
                 print(
                     separator.join(
                         (str(datetime.now()),
+                         str(args.ip),
```

Comment:
IP address is logged on line #41 and it is not expected to change in a loop. Do we really need it repeated every line?

Reply:
It's just because if you have more devices and you open a few instances, then you can lose track of the devices.
Maybe it's worth to add an input parameter to make this trace optional.